spec/core/data_structures.md

```diff
@@ -455,8 +466,8 @@ func SumTruncated(bz []byte) []byte {
 
```

Comment:
Suggestion: "This parameters are used by the Proposed-Based TimeStamp (PBTS) algorithm", link to https://github.com/tendermint/tendermint/tree/master/spec/consensus/proposer-based-timestamp